documentation/tutorial/02-environment.md

```diff
@@ -13,75 +13,40 @@ Before we start, make sure your machine meets the following requirements:
   [Multipass](https://multipass.run/docs). A simple set of instructions to set
   up Multipass can be found
   [here](https://juju.is/docs/sdk/set-up-your-development-environment#heading--set-up-an-ubuntu-vm-with-multipass).
-  Please keep in mind that the instructions in this tutorial have not been
-  tested with Multipass.
 - 8GB of RAM.
 - 2 CPU threads.
 - At least 20GB of available storage.
 - Access to the internet for downloading the required snaps and charms.
 
-## Install MicroK8s
+## Install Kubernetes and bootstrap Juju
 
-On your machine, install and configure MicroK8s:
+Charmed Temporal is deployed to Kubernetes using Juju. Before deployment, you need:
 
-```bash
-# Install Microk8s from snap:
-sudo snap install microk8s --channel 1.25-strict/stable
-
-# Add your user to the MicroK8s group:
-sudo usermod -a -G snap_microk8s $USER
+1.  **A running Kubernetes cluster**:  Canonical K8s is our preference. See [Canonical Kubernetes documentation](https://documentation.ubuntu.com/canonical-kubernetes/release-1.32/snap/howto/install/snap/) for installation instructions.
 
-# Give your user permissions to read the ~/.kube directory:
-sudo chown -f -R $USER ~/.kube
+2. **Juju bootstrapped to your cluster**. See [Get started with Juju](https://documentation.ubuntu.com/juju/3.6/tutorial/) for more details.
 
-# Create the 'microk8s' group:
-newgrp snap_microk8s
+[note]
 
-# Enable the necessary MicroK8s addons:
-sudo microk8s enable hostpath-storage dns
-
-# Set up a short alias for the Kubernetes CLI:
-sudo snap alias microk8s.kubectl kubectl
-```
+Our recommendation is bootstrapping Juju on a Canonical K8s cluster. See [Canonical K8s cloud and Juju](https://documentation.ubuntu.com/juju/3.6/reference/cloud/list-of-supported-clouds/the-canonical-ks-cloud-and-juju/) and the [bootstrap command reference](https://documentation.ubuntu.com/juju/3.6/reference/juju-cli/list-of-juju-cli-commands/bootstrap/) for more details.
 
-[MicroK8s](https://microk8s.io/docs) is a minimal production Kubernetes, so now
-you have a small Kubernetes cloud (by default called microk8s) on your machine.
+[/note]
 
-## Set up Juju
+## Create the Temporal Model
 
-On your machine, install Juju, connect it to your MicroK8s cloud, and prepare a
-workspace (‘model’):
+Once you have Juju bootstrapped to your Kubernetes cluster, create a workspace, or 'model', for Temporal. You can name it `temporal-model`:
```

Comment:
I just noticed this phrase. Since we are in Juju world, let's just call it `model`.

```suggestion
Once you have Juju bootstrapped to your Kubernetes cluster, create a `model`, for Temporal. You can name it `temporal-model`:
```